pages/components/rating.html

```diff
@@ -106,8 +106,35 @@ <h1 clss="page-title">Ratings</h1>
 
 
   <main class="main-content">
-
-    <iframe src="/codeblock/x.txt" frameborder="0" class="codeblock"></iframe>
+    <div class="rating-wrapper wrap-color">
+        <input type="radio" name="rating" id="star-1">
+        <label for="star-1">✫</label>
```

Comment:
not sure about this, but i think using an icon for star would be better than typing it? I guess it's fine maybe

Reply:
Icons may take time to load. this is unicode value like &nbsp; or &gt;